src/sentry/workflow_engine/handlers/action/notification/issue_alert.py

```diff
@@ -401,11 +402,20 @@ def get_target_identifier(
             }
         raise ValueError(f"No target identifier key found for action type: {action.type}")
 
+    @classmethod
+    def process_settings(cls, settings: list[dict[str, Any]]) -> list[dict[str, Any]]:
+        # Process each setting, removing None labels
+        return [
+            {k: v for k, v in setting.items() if not (k == "label" and v is None)}
+            for setting in settings
+        ]
+
     @classmethod
     def get_additional_fields(cls, action: Action, mapping: ActionFieldMapping) -> dict[str, Any]:
         # Need to check for the settings key, if it exists, then we need to return the settings
         # It won't exist for legacy webhook actions, but will exist for sentry app actions
         if action.data.get("settings"):
-            blob = SentryAppDataBlob(**action.data)
-            return {"settings": blob.settings}
+            blob = asdict(SentryAppDataBlob(**action.data))
```

Comment:
I'm not sure why passing this through `asdict` is required when it wasn't before? don't we already have access to the `settings` property of `SentryAppDataBlob`?